educator.rb

```diff
@@ -0,0 +1,41 @@
+class Educator 
+  attr_reader :age, :salary, :phase, :target_raise
+  attr_accessor :name
```

Comment:
Some of these attrs and lines in initialize look awfully similar to the ones here... how can you refactor?